queries/leo/highlights.scm

```diff
@@ -0,0 +1,180 @@
+(variable) @variable
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
```

Comment:
These could be `@exception`

Reply:
We should make this consistent at some point.
Many languages don't highlight asserts as `@exception`.